tests/generation/BearsTest.py

```diff
@@ -158,8 +161,10 @@ def test_filter_relevant_bears_green_mode(self):
         for key in ['Python', 'C', 'All']:
             bear_obj_dict[key] = set()
             for bear in IMPORTANT_BEAR_LIST[key]:
-                bear_obj_dict[key].update([x for x in bear_objs if (
```

Comment:
revert to this line

exactly

https://github.com/coala/coala-quickstart/pull/290#pullrequestreview-145474145